scripts/convert_weights_for_gaudi2.py

```diff
@@ -82,14 +82,20 @@ def convert_files(input_path, output_path):
                 tensor = tensor_file.get_tensor(k)
                 if "proj" in k:
                     if k.endswith("weight"):
-                        tensor = (tensor.float() * 0.5).to(
+                        tensor = (tensor.float() * 240.0 / 448.0).to(
                             torch.float8_e4m3fn
                         )
+                    elif k.endswith("weight_scale") or k.endswith(
+                        "input_scale"
+                    ):
+                        tensor = tensor.float() * 448.0 / 240.0
```

Comment:
suggest to change those hard-coded number (448, 240, etc.) to some micro or torch.info if possible.

Reply:
fixed